src/features/vote/CommitPhase.tsx

```diff
@@ -184,9 +184,11 @@ const CommitPhase: FC<Props> = ({
       const identifiers = Object.keys(watchAllFields);
       const values = Object.values(watchAllFields);
       for (let i = 0; i < identifiers.length; i++) {
-        if (values[i] !== "") {
+        if (values[i] !== "" && values[i] !== undefined) {
+          const idenSplit = identifiers[i].split("~");
+
           const val = {
-            identifier: identifiers[i],
+            identifier: `${idenSplit[0]} -- ${idenSplit[1]}`,
```

Comment:
Can you explain what this is doing?

Reply:
Purely visual. This is to show the identifier and timestamp from the name of the input.

It's a way of abstracting the data without doing a look up in the array of values.

Reply:
Going to confirm some stuff though.

Reply:
I see. Thanks!